medusa/post_processor.py

```diff
@@ -490,8 +455,9 @@ def _int_move_and_sym_link(cur_file_path, new_file_path):
                           (cur_file_path, new_file_path, e), logger.ERROR)
                 raise
 
+        subtitle_action = move if self.process_method in ['hardlink', 'symlink'] else eval(self.process_method)
         self._combined_file_operation(file_path, new_path, new_base_name, associated_files,
-                                      action=_int_move_and_sym_link, subtitles=subtitles)
+                                      action=eval(action), subtitle_action=subtitle_action, subtitles=subtitles)
```

Comment:
I need to read your code on a desktop. But I suspect your using eval here, while that's not needed. You can also map the different methods in a dict.